tests/fetch_data/public-keys

```diff
@@ -0,0 +1,3 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC/rGkslgprWZ/J+k7Uu2h0zS5GVwO4P13fkSbtcMKDjmMSffUWKeCM/z8XRgDcpr7iBWsG8eUGdaoxAo0St+w69pFZCTfVFmOmkNQezUnw5JlvmSIKhVTLwUHYO+K6trBLoBDHVf/O5IGN6burKswO5VqCj6wHEkR2gPKZ0UxF57ll4a4Q3NQRwPjkmrjtUCINlStXgpVbCbNSrJ8i93LPPZTZeMkdhvqswNvxzWRjJhy5QX7ScvJXDd9WNtU5kzT60cq9W+hgMVNHBz0cql/ig2vrQSqS0kPbJ+8gggz9agnE19XmgvZ5hZjW+hdmwrQgiP2DY7L2JfAdTQR5o1nHHmpUeTThjCka8VxEWrgFM8fpCkl8zYhTBXh5vc6xqUnkkRkL10smbOZ2SD5GEV+a8gXRTgvJeBorj95rSJRbfklxHpYCSfC0mT3r4eLhqAmnwjQSwVZbOiDLvD8KGFBWShz2AmWA/+0M736eGMvczNagSr+M1oK8uLfzi1XEE80= test@test-system
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCnluX37ew5kLnsyvgmTffPhnjsyU0im7bV9CNbF82aGBxtsvgeumINdgvVp/kTXxIbWQyH6ys/mygNYOtTxLIUFR2yj5aAzjxrT0bLRL1i1WI7JbPjgMcl3WsHWGZunpqTYy+Ip/0h2p2+teMOO6avQ69mMSIDCirRzMaO2dovtyOTmLZfPeGd2j+uDLMCyFwjnh0WOCd+RbkohgA0iC3hLnoVhanLrV9wnQiAn6W/rSgF9Jt50qLvS2xFHJGNdS9u2+RZ0P0WaLsSADmyu235vHnfLvG/9rfrqbzVEGdFLhUvS9yz4QvLWkdPkxojllof/LMAubb/Q/zTZbrZjHGgFYNlbodn6NaD+5u9csgafrjuS39RFnoar1SRpoeXe6aL3noo7LoniRjuyc+Jjbx6kUcN0v8gCzMmToXqlITs8qD7kUSmai3GF6sETGEMcF0aJ5tPyq/95vEQSBBc9JCmFR4VtZ/XyxHmL3WNRiiCgCL0SOFvliGVb5fRByN6CUc= test@test-system
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDTJyajEZTl1EEVBUE1EmnjTlGV2cqixLgw0ogEItsP5bl3AerPHEYO6rTt2IIh2hbNM5VAiPW1A3Fp/MnnICwHxTi3C/EnAwE6m6fc0mq/VBgyZWCqDiTdKBeyIu7Zg5Vpzyo8whiCA/i3jme45ED+/FmKlM1v4HtbaOXezxnlLEEbw3WfjFJFH6zKIkI3MvYVmKn3zNVEDds0WQRRbfI1LPlRhod42TJ+rWgxVe5W+zwsl/Ctc4ShreSIbUggMAHKvLe1bWreAGT0jUpDgBgM6bj2EZbhYpsD7deg/xV9ZgboU5jsnSnd2uFxKgKf3GNvQGJII1hQuwphCM35PSKSQCKbzQH5+CrWuPuhUyCFUOqUvmmzS2dZu6YPOe6bO+orks3uHNpMFt3o82w0t+07ehL9eqNc+rAqKZ/beYyCKmuioC/gwIra5YgPbhMvV+0mEMEzD+UnJ7gx8/EvaTUgdwJdjgPSakoxU+0r6u5FwOo2LxGia6sQF/JgXX3192E= test@test-system
```

Comment:
I wouid just put `SSH_TEST_KEY_STRING_1` `.._2` in here but not any actual key. Then validate the output.

Reply:
Converted the data. Just need to add a test.

Reply:
Test added